File: Daily_Practice/Swap_Two_numbers_without_using_third_variable.py
var1 = 10
var2 = 20

print(f'Before swapping var1 = {var1} & var2 = {var2}')

var1 = var1 - var2
var2 = var1 + var2
var1 = var2 - var1

print(f'After swapping var1 = {var1} & var2 = {var2}')

File: Daily_Practice/Swap_Two_numbers_without_using_third_variable.go
package main

import "fmt"

func main() {
	var num1 = 10
	var num2 = 20

	fmt.Printf("Before Swapping num1 = %d & num2 = %d\n", num1, num2)

	num1 = num1 - num2
	num2 = num1 + num2
	num1 = num2 - num1

	fmt.Printf("After Swapping num1 = %d & num2 = %d", num1, num2)
}
